### PR TITLE
Tox testing jobs

### PR DESCRIPTION
**Pull request by stevekuznetsov**

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Python 3.5 are currently nonfunctional until we figure out the SCL bits for them. That's ok -- same as today with Jenkins.

/cc @sosiouxme @csrwng

File: projects/test-bases/openshift/openshift-ansible.yaml [hunk 1]
```diff
@@ -20,13 +20,14 @@ items:
...
         FROM openshift/origin-release:base
-        RUN INSTALL_PKGS="gcc libffi-devel python-devel openssl-devel python2-pip" && \
+        RUN INSTALL_PKGS="ansible gcc libffi-devel python-devel openssl-devel python2-pip" && \
             yum install -y $INSTALL_PKGS && \
             rpm -V $INSTALL_PKGS && \
             yum clean all && \
             pip install tox
         ENV GIT_COMMITTER_NAME=developer GIT_COMMITTER_EMAIL=developer@redhat.com
         RUN umask 0002 && mkdir -p /data/src/github.com/openshift/ && git clone https://github.com/openshift/openshift-ansible.git /data/src/github.com/openshift/openshift-ansible/
+        RUN umask 0002 && cd /data/src/github.com/openshift/openshift-ansible/ && /usr/bin/tox --notest
```
Comment (stevekuznetsov) [inline projects/test-bases/openshift/openshift-ansible.yaml]: With this the `tox` refresh per-PR takes <1min

**Comment by csrwng** [comment]
/lgtm
@stevekuznetsov there's got to be a better way to define the jobs in jenkins than copying/pasting the same yaml over and over again. Maybe a smarter controller ?

**Comment by sosiouxme** [comment]
@stevekuznetsov what's the plan for the integration tests?

**Comment by stevekuznetsov** [comment]
@csrwng whether we autogen the JJB and check it in as a CM or check in a CM with a list of Jenkinsfiles and have a controller that auto-gens a JJB for each ... not sure one is much better than the other. For the time being, meh.

**Comment by stevekuznetsov** [comment]
@sosiouxme not really sure today. @csrwng FYI their integration tests spin up loads of containers and test against them as target hosts for playbooks. I think we'll need to launch a VM for that?

**Comment by stevekuznetsov** [comment]
Merging so we can build on top

**Comment by stevekuznetsov** [comment]
man i love robots

**Pull request merged**